src/main/webapp/guestbook.xhtml

```diff
@@ -197,4 +196,4 @@
             </ui:define>
         </ui:composition>
     </h:body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Weird. Couldn't tell what, if anything, changed on this line. Same with the template.xhtml file.